README.md

```diff
@@ -23,9 +26,9 @@
 [repostatus-url]: https://www.repostatus.org/#active
 [repostatus-img]: https://www.repostatus.org/badges/latest/active.svg
 
-| **Documentation** | **Tests** | **Coverage** | **How to contribute?** | **Project status** |
-|:--------------:|:-------:|:---------:|:---------:|:---------:|
-| [![docs-img-stable]][docs-url-stable] [![docs-img-dev]][docs-url-dev] | [![CI][ci-img]][ci-url] | [![codecov][cov-img]][cov-url] | [![contrib][contrib-img]][contrib-url] | [![repostatus-img]][repostatus-url] |
+| **Documentation** | **Docker** | **Tests** | **Coverage** | **How to contribute?** | **Project status** |
+|:---:|:---:|:---:|:---:|:---:|:---:|
```

Comment:
Can we also add the link to `apptainer`?

Reply:
maybe separate list/table could do

Reply:
Converted to a separate subsection